include/swift/ABI/Task.h

```diff
@@ -333,14 +333,15 @@ class AsyncTask : public Job {
 
   // ==== Task Local Values ----------------------------------------------------
 
-  void localValuePush(const HeapObject *key,
+  void localPushValue(const HeapObject *key,
                       /* +1 */ OpaqueValue *value,
                       const Metadata *valueType);
 
-  OpaqueValue *localValueGet(const HeapObject *key);
+  void localPushStop();
```

Comment:
I'd like to use the term `Barrier` for those; so following the existing naming it would be `localValuePushBarrier`

Reply:
Done